Clean Swift/Scene.xctemplate/UICollectionViewController/___FILEBASENAME___Worker.swift

```diff
@@ -8,11 +8,9 @@
 
 import UIKit
 
-class ___FILEBASENAMEASIDENTIFIER___Worker {
+protocol ___FILEBASENAMEASIDENTIFIER___Worker {
```

Comment:
Worker should stay class but should conform to ___FILEBASENAMEASIDENTIFIER___WorkerProtocol